website/docs/language/tests/mocking.mdx

```diff
@@ -171,9 +181,9 @@ Overrides can be used with both real and mocked providers and will provide the c
 
 ### Overrides Syntax
 
-All override blocks contain a `target` attribute, which should specify the resource, data source, or module to override. The `override_module` blocks contain an `outputs` attribute, while the `override_resource` and `override_data` blocks contain a `values` attribute.
+All override blocks contain a `target` attribute, which should specify the resource, data source, or module to override. The `override_module` blocks contain an `outputs` attribute, while the `override_resource` and `override_data` blocks contain a `values` attribute. In addition, the override blocks also support the same `override_during` attribute seen within the `mock_provider` block.
 
-The `outputs` and `values` attributes are optional and if not specified, Terraform will generate values for them automatically.
+The `outputs` and `values` attributes are optional and if not specified, Terraform will generate values for them automatically. The `override_during` attribute is also optional, and if specified it will override the `override_during` attribute in the `mock_provider` block. If not specified, the behavior will be inherited from the `mock_provider` block.
```

Comment:
```suggestion
The `outputs` and `values` attributes are optional and if not specified, Terraform will generate values for them automatically. The `override_during` attribute is also optional, and if specified it will override the `override_during` attribute in the `mock_provider` block. If not specified, Terraform inherits the behavior from the `mock_provider` block.
```